Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM silintl/php7:7.2
+FROM silintl/php7:7.4
 
-MAINTAINER Phillip Shipley <phillip.shipley@gmail.com>
+LABEL maintainer "Phillip Shipley <phillip.shipley@gmail.com>"
 
-ENV REFRESHED_AT 2020-04-16
+ENV REFRESHED_AT 2021-06-14
```

Comment:
Heh. It's not worth changing, but did we all miss that the date/month was wrong or is it based on when you started working on the change?  :-)

Reply:
Sadly, that's probably when I started working on this change... over a month ago